### ISSUE TITLE
No documentation for rotateX, rotateY and rotateZ

### ISSUE THREAD

**Issue by nkolban** [processing/p5.js-website#257]
I am unable to find any documentation on rotateX, rotateY and rotateZ methods.

**Comment by limzykenneth** [comment]
As far as I can remember (not very far tbh) there's only [`rotate`](http://p5js.org/reference/#p5/rotate), [`rotationX`](http://p5js.org/reference/#p5/rotationX), [`rotationY`](http://p5js.org/reference/#p5/rotationY) and [`rotationZ`](http://p5js.org/reference/#p5/rotationZ)...

**Comment by indefinit** [comment]
in webgl mode, there is rotateX, rotateY, and rotateZ methods.  It looks like they aren't being pulled into the docs though because I forgot to update those function comments.  I'll go ahead and do that now in the library

FWIW:

This one's my favorite method since it's a convenience function for rotating around each of the axes.  The trick is that you'll need a normalized array representing the rotational vector.
```
/**
 * [rotate description]
 * @param  {Number} rad  angle in radians
 * @param  {p5.Vector | Array} axis axis to rotate around
 * @return {p5.RendererGL}      [description]
 */
rotate(angle, [1,0,0]); 
```


```
/**
 * [rotateY description]
 * @param  {Number} rad rad radians to rotate
 * @return {[type]}     [description]
 */
rotateY(angle);
```

```
/**
 * [rotateZ description]
 * @param  {Number} rad rad radians to rotate
 * @return {[type]}     [description]
 */
rotateZ(angle);
```

**Comment by lmccart** [comment]
closed with #1455
